### DISCUSSION TITLE
transferring  between ArchR and Signac

### DISCUSSION THREAD

**Discussion by pengxin2019** [GreenleafLab/ArchR#934] [Questions / Documentation]
Hello,
I did not find ArchR until yesterday. Thanks for developing this wonderful tool which .

There are  two questions below before I make the decision on how to use it in my research:
1. Do you think it possible to remove the doublet by running createArrowFiles ( using the fragment produced by cell ranger atac as the input) , addDoubletScores  and filterDoublets and then transfer the filtered files to Signac? If it is doable, which file should be used to go to Signac/Seurat or in other words how to do it?
If I understand correctly, filterDoublets would remove some cells that are predicted as doublets, if I can collect the removed cell names (16bp barcode of 10x Genomics), then I would be able to know which cells should be removed from Signac object, right?  
2. It is really cool that ArchR can integrate scRNA and scATAC. But I just want to double check that scRNA and scATAC datasets could be collected from independent cells? I know scATAC could be processed by ArchR. But in terms of the analysis of scRNA before it is integrated with scATAC, we should perform QC etc using other tools, right?
Thanks
Best
Penny

**Comment by rcorces** [comment]
1. There shouldnt really be too much need to go between ArchR and Signac - if there is a key feature that Signac has that you think ArchR is missing, just let us know. That being said, doublet scores are stored in `cellColData`.
2. Yes - ArchR works for ATAC and RNA data acquired from different cells (see the manual) as well as the newer multiome approaches. Our expectation is that you process your RNA-only data outside of ArchR (for ex. with Seurat) and then use that object as import to ArchR.
